index.js

```diff
@@ -11,6 +11,10 @@ const tutorials = [
   'what is JSONP?'
 ];
 
-const titleCased = () => {
-  return tutorials
-}
+const titleCased = () => tutorials.map(removeSpaces)
```

Comment:
`removeSpaces` is misleading. Yes, it does remove spaces, in a way, but it does so much than that, doesn't it?

Reply:
Yeah, it removes the spaces between each word and creates an array where each word is an item in an array. Maybe something like createWordArray?